usefulUtilities/modelJSONCleanup/modelJSONCleanup.js

```diff
@@ -0,0 +1,183 @@
+            return true;
+        }
+    }
+
+    // Entities with Parents
+    if (ent.parentID) {
+        let parentID = ent.parentID;
+        let parent = parentMap[parentID];
+        if (!parent) {
+            countNoParentInDomain++;
+            // Entity has a parentID, but there is no parent in the domain so delelte it
+            return false;
+        }
+        if (parent.position) {
+            // Delete anything with a parent that is in bounds
+            if (
+                (parent.position.x < CONFIG.MAX_X && parent.position.x > CONFIG.MIN_X) &&
+                (parent.position.y < CONFIG.MAX_Y && parent.position.y > CONFIG.MIN_Y) &&
+                (parent.position.z < CONFIG.MAX_Z && parent.position.z > CONFIG.MIN_Z)
+            ) {
+                countParentInBound++;
+                return true;
+            } else {
+                countParentOutBound++;
+                return false;
+            }
+        }
+    } else if (ent.position) {
+    // Entities without Parents
+
+            // Delete anything with a parent that is in bounds
+            if (
+                (ent.position.x < CONFIG.MAX_X && ent.position.x > CONFIG.MIN_X) &&
+
+                (ent.position.y < CONFIG.MAX_Y && ent.position.y > CONFIG.MIN_Y) &&
+                (ent.position.z < CONFIG.MAX_Z && ent.position.z > CONFIG.MIN_Z)
+            ) {
+                countInBound++
+                return true;
+            } else {
+                countOutBound++
+                return false;
+            }
+    }
+
+    // Otherwise keep the entity
+    countRemainingKeeps++;
+    return true;
+})
+
+// Stats
+console.log("countNameBlackListGroup: ", countNameBlackListGroup);
+console.log("countNameWhiteListGroup: ", countNameWhiteListGroup);
+console.log("countPatternMatchBlackListGroup: ", countPatternMatchBlackListGroup);
+console.log("countPatternMatchWhiteListGroup: ", countPatternMatchWhiteListGroup);
+console.log("countTypeBlackListGroup: ", countTypeBlackListGroup);
+console.log("countTypeWhiteListGroup: ", countTypeWhiteListGroup);
+console.log("countInBound: ", countInBound);
+console.log("countOutBound: ", countOutBound);
+console.log("countParentInBound: ", countParentInBound);
+console.log("countParentOutBound: ", countParentOutBound);
+console.log("countNoParentInDomain: ", countNoParentInDomain);
+console.log("countRemainingKeeps: ", countRemainingKeeps);
+console.log("\nFinal ent Length: ", modelJSONEntities.length, "\n");
+
+// Sum of deletes
+var deletes = [countOutBound, countParentOutBound, countNameBlackListGroup, countTypeBlackListGroup, countNoParentInDomain, countPatternMatchBlackListGroup].reduce( (prev, curr) => {
+    return prev += curr;
+}, 0);
+console.log("total Deletes: ", deletes);
+// Sum of Keeps
+var keeps = [countInBound, countParentInBound, countNameWhiteListGroup, countTypeWhiteListGroup, countPatternMatchWhiteListGroup, countRemainingKeeps].reduce( (prev, curr) => {
```

Comment:
move to two lines and also
```suggestion
var keeps = [countInBound, countParentInBound, countNameWhiteListGroup, countTypeWhiteListGroup, countPatternMatchWhiteListGroup, countRemainingKeeps].reduce((prev, curr) => {
```